### PR TITLE
Allow SonataBlockBundle 3.0

### LINKED ISSUE #122: Add support for BlockBundle 3.0
Sonata admin 3.0 requires BlockBundle 3.0. This means the seo and admin bundle can't be installed together on Symfony 3 atm.

### PR DESCRIPTION
**Pull request by WouterJ**
Fixes #122 

I can't see any BC breaks between BlockBundle 2.3.11 and master, so I think 3.0-dev is already supported.

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
The idea is to push major on most bundles to start with a new one but seo-bundle 2.0 looks pretty recent so we could make an exception.

Also, block bundle 3.0 is gonna be released soon. I'll ping you for a rebase after that.

**Comment by Soullivaneuh** [comment]
Block bundle 3.0 is out and Travis is not started yet. This should be OK so.

File: composer.json [hunk 1]
```diff
@@ -17,18 +17,18 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/options-resolver": "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.3 || ~3.0",
```
Comment (core23) [inline composer.json]: We should change this to ``"symfony/framework-bundle": "^2.3 || ^3.0"``

Comment (WouterJ) [inline composer.json]: Let's do that in another PR?

Comment (Soullivaneuh) [inline composer.json]: Well, since you changed the constraint notation, I think we can have this modification too on this PR.

**Comment by Soullivaneuh** [comment]
Closed for #124.
